tests/frontend/helper.js

```diff
@@ -100,10 +115,16 @@ var helper = {};
     $iframeContainer.find("iframe").purgeFrame().done(function(){
       $iframeContainer.append($iframe);
       $iframe.one('load', function(){
+        helper.padChrome$ = getFrameJQuery(                $('#iframe-container iframe'));
+        if (opts.clearCookies) {
+          helper.clearPadPrefCookie();
+        }
+        if (opts.padPrefs) {
+          helper.setPadPrefCookie(opts.padPrefs);
+        }
         helper.waitFor(function(){
           return !$iframe.contents().find("#editorloadingbox").is(":visible");
         }, 50000).done(function(){
-          helper.padChrome$ = getFrameJQuery(                $('#iframe-container iframe'));
```

Comment:
Whereas the other two seem to require the page to have loaded, the iframe itself doesn't. And that's great, because now we can grab it ahead of time and set the prefs cookie way sooner. Soon enough for my ep_webrtc test anyway.